addons/interact_menu/functions/fnc_collectActiveActionTree.sqf

```diff
@@ -43,12 +43,14 @@ _origActionData params [
     "_distance"
 ];
 
+private _result = [_target, ACE_player, _customParams] call _conditionCode;
+
```

Comment:
Could log bad returns to help addons fix their code
Trees are cached so this should be spammed that much
```suggestion
if (isNil "_result") then { ERROR_1("Action [%1] bad condition return",_actionName); _result = false; };
```